templates/data/opensearch/support.html

```diff
@@ -0,0 +1,541 @@
+{% extends 'base_index.html' %}
+
+{% block title %}
+  Get support for OpenSearch, an open source search engine, analytics tool and vector database.
+{% endblock %}
+
+{% block meta_description %}
+  Get support for OpenSearch, from Canonical. 24/7 or weekday support and up to ten years security patching per release
+  track.
+{% endblock %}
+
+{% block meta_copydoc %}
+  https://docs.google.com/document/d/1ovtXv8RBybw-flvttyqWOzZivUryQMP34T1UGe90HlY/edit
+{% endblock meta_copydoc %}
+
+{% block body_class %}
+  is-paper
+{% endblock body_class %}
+
+{% block content %}
+  <section class="p-section--hero is-shallow u-no-padding--bottom">
```

Comment:
Make it just a p-section--hero, no additional classes needed

Reply:
I would have but the spacing becomes more than that is in design
![image](https://github.com/user-attachments/assets/f1ac725e-2548-41bb-bda2-4872c7a5a0b2)
![image](https://github.com/user-attachments/assets/cc69117c-1dd7-4db7-91a4-d1d7dc4a17b7)